components/Questionnaire/Two/index.vue

```diff
@@ -9,13 +9,34 @@
           <p class="mb-3">
             Komunitas apa saja yang saat ini ada di Desa Bapak/Ibu? (Boleh pilih dari satu)
           </p>
-          <jds-checkbox-group
+          <label v-for="(item, index) in communities" :key="index" class="custom-checkbox">
+            {{ item.value }}
+            <input
+              v-model="literasi_digital.komunitas.data"
+              type="checkbox"
+              name="community-list"
+              :value="item.value"
+              @change="onCommunityListSelected"
+            >
+            <span class="checkmark" />
+          </label>
+          <label class="custom-checkbox">
+            Tidak ada komunitas
+            <input
+              type="checkbox"
+              name="community-list-none"
+              value="Tidak ada komunitas"
+              @change="onCommunityListNoneSelected"
+            >
+            <span class="checkmark" />
+          </label>
+          <!-- <jds-checkbox-group
             v-model="literasi_digital.komunitas.data"
             :options="communities"
             value-key="value"
             label-key="value"
-          />
-          <div class="grid grid-cols-5 mt-4">
+          /> -->
```

Comment:
unused code?

Reply:
Thanks kak @yoslie, it's already fix in [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/96/commits/a3b5eac26b324f2579422819d170591ba7e0786f)